docs/sources/installation/configuration.md

```diff
@@ -350,7 +350,7 @@ Default is `false`.
 
 ### data_source_proxy_whitelist
 
-Define a white list of allowed ips/domains to use in data sources. Format: `ip_or_domain:port` separated by spaces.
+Define a whitelist of allowed ips/domains to use in data sources. Format: `ip_or_domain:port` separated by spaces. This feature is currently not available for PostgreSQL, MySQL, and MSSQL data sources.
```

Comment:
Tried to make it clearer, but not sure I accomplished that. 

```suggestion
Define a whitelist of allowed IP addresses or domains to be used as data source URLs and proxied through the Grafana data source proxy. Format: `ip_or_domain:port` separated by spaces. This setting doesn't apply to  PostgreSQL, MySQL, MSSQ data sources because they are not being proxied through the Grafana data source proxy.
```

Reply:
@marefr s/MSSQ/MSSQL/ :)

Reply:
Updated suggestion

Reply:
Isn't it better also to keep the wording "PostgreSQL, MySQL, and MSSQL data sources" also?

Reply:
Please add your suggestion, I'm not exactly sure what you mean. Thanks

Reply:
I'm not sure the technical details of the grafana data source proxy is needed to explain the property clearly to users.
But now I'm also wondering if the property would be useless if the data source mode is set to Browser instead of server. This might be interesting to write down too.

Reply:
@marefr I mean, I think it's better to keep the wording "PostgreSQL, MySQL, and MSSQL data sources".

Reply:
How about this:

```suggestion
Define a whitelist of allowed IP addresses or domains to be used as data source URLs with the Grafana data source proxy. Format: `ip_or_domain:port` separated by spaces. PostgreSQL, MySQL, and MSSQL data sources  do not use this proxy and are therefore unaffected by this setting
```
I believe it is a little easier to read as it avoids repeating the `proxy` word and its variations 4 times in two sentences.

Reply:
LGTM. @aknuds1 what do you think?

Reply:
LGTM except I would write "do not use the proxy" instead of "do not use this proxy", since it sounds weird with the following "this setting".

Reply:
@aknuds1 Fixed using your suggestion. Are we good to go? Do the 3 commits need to be merged together?